src/test/java/org/openstreetmap/atlas/checks/validation/areas/WaterbodyAndIslandSizeTest.java

```diff
@@ -0,0 +1,94 @@
+package org.openstreetmap.atlas.checks.validation.areas;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * WaterbodyAndIslandSizeTest unit tests
+ *
+ * @author danielbaah
+ */
+public class WaterbodyAndIslandSizeTest
+{
+    @Rule
+    public WaterbodyAndIslandSizeTestRule setup = new WaterbodyAndIslandSizeTestRule();
+
+    @Rule
+    public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
+
+    private final WaterbodyAndIslandSizeCheck check = new WaterbodyAndIslandSizeCheck(
+            ConfigurationResolver.emptyConfiguration());
```

Comment:
This check is not stateless (because it is flagging identifiers). So the order of tests could change the result.